examples/display/main/esp_logo.c

```diff
@@ -1,423 +0,0 @@
-    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x65, 0xe9, 0x0c, 0xa5, 0xe9, 0xa3, 0xa5, 0xe9, 0xff, 0xa5, 0xe9, 0xff, 0xa5, 0xe9, 0xff, 0xa5, 0xe9, 0xff, 0xa5, 0xe9, 0xff, 0xa5, 0xe9, 0xdc, 0xa5, 0xe9, 0x4c, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xa6, 0xe9, 0x33, 0xa6, 0xe9, 0x73, 0xa5, 0xe9, 0xab, 0xa5, 0xe9, 0xe3, 0xa5, 0xe9, 0xff, 0xa5, 0xe9, 0xff, 0xa5, 0xe9, 0xff, 0xa5, 0xe9, 0xff, 0xa5, 0xe9, 0xfc, 0xa6, 0xe9, 0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xab, 0xfa, 0x03, 0xa5, 0xe9, 0x34, 0xa5, 0xe1, 0x13, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0
```

Comment:
The file genereated for LVGL 8 only contains `#if LV_COLOR_DEPTH == 16 && LV_COLOR_16_SWAP == 0`. 

E.g. on Kaluga board this will resolve to empty array as we set `LV_COLOR_16_SWAP`

Reply:
Thank you for check this. I overlooked that there must be param for SWAP. I will add it and test it. Thank you!

Reply:
Fixed.